libc/rest/Makefile

```diff
@@ -0,0 +1,10 @@
+LOCAL_DIR := $(call my-dir)
+PARENT_DIR := $(shell dirname $(realpath $(LOCAL_DIR)))
```

Comment:
I think it could be better

```suggestion
PARENT_DIR = $(shell dirname $(realpath $(LOCAL_DIR)))
```

Reply:
nah, always use `:=` in makefiles if possible (this makes variable value to be computed once instead at every usage).

Buut... the `Makefile` should not try compiling anything in `..`, the common files should probably be made into static lib to be reused in sub-components if needed.

Reply:
@nalajcie So, do you think the name `libtest` would be adequate for this static library?